Makefile

```diff
@@ -128,6 +128,10 @@ help: ## Print this help
 
 ##@ Build 🔨
 
+.PHONY: build-analyzer-executor
+build-analyzer-executor:
+	./pants package ./src/python/analyzer_executor/src
```

Comment:
We can probably just fold this into `build-lambda-zips-python` instead of a bespoke makefile target, no?

Reply:
It's not a lambda

Reply:
oh duh yeah